examples/emu3.5/conf/serve/emu3p5.yaml

```diff
@@ -0,0 +1,9 @@
+- serve_id: A
+  module: examples/emu3.5/conf/serve/emu3p5.py
+  name: Emu3_5InferencePipeline 
+  resources:
+    max_ongoing_requests: 1
+    target_ongoing_requests: 1
+    num_gpus: 2
+    min_replicas: 1
+    max_replicas: 2
```

Comment:
What is the relationship between num_gpus, max_replicas and tensor-parallel-size ?

Reply:
max_replicas means instance scaling limit num,
num_gpus is the same as tensor-parallel-size.